taxi/models.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.name} ({self.country})"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255)
+
+    class Meta:
+        constraints = [UniqueConstraint(
+            fields=["license_number"],
+            name="unique_license_number")]
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name} ({self.username})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(to=Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ForeignKey(to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Pls check the scheme:
https://user-images.githubusercontent.com/80070761/159295912-d02c7080-09a7-41ec-aa86-b0ae3afdd75b.png
You need create ManyToMany field:
`drivers = models.ManyToManyField(to=settings.AUTH_USER_MODEL, related_name="cars")`

Reply:
Thanks, fixed!